products/dns/api.yaml

```diff
@@ -47,6 +47,8 @@ objects:
           A mutable string of at most 1024 characters associated with this
           resource for the user's convenience. Has no effect on the managed
           zone's function.
+        update_verb: :PATCH
```

Comment:
Oh sorry, it has support for PUT as well, which is the default. I'd just leave this and the line after out.

Reply:
And then change the `input: true` on the resource so that it generates an update.